draft-ietf-quic-tls.md

```diff
@@ -942,16 +942,28 @@ Packet number protection is applied after packet protection is applied (see
 encryption algorithm.
 
 In sampling the packet ciphertext, the packet number length is assumed to be the
-smaller of the maximum possible packet number encoding (4 octets), or the size
-of the protected packet minus the minimum expansion for the AEAD.  For example,
-the sampled ciphertext for a packet with a short header can be determined by:
+smaller of the maximum possible packet number encoding (4 octets), or the
+remaining space in the packet when the minimum expansion for the AEAD is
+subtracted.  For example, the sampled ciphertext for a packet with a short
+header can be determined by:
 
 ~~~
-sample_offset = min(1 + connection_id_length + 4,
+sample_offset = min(1 + len(connection_id) + 4,
                     packet_length - aead_expansion)
 sample = packet[sample_offset..sample_offset+sample_length]
 ~~~
 
+A packet with a long header is sampled in the same way, noting that multiple
+QUIC packets might be included in the same UDP datagram and that each one is
+handled separately.
+
+~~~
+sample_offset = min(2 + len(destination_connection_id) +
```

Comment:
Shouldn't `2` be `6`? There are the type octet, version (4 octets), CILs (1 octets).